service/header/interface.go

```diff
@@ -33,10 +33,6 @@ type Broadcaster interface {
 // Exchange encompasses the behavior necessary to request ExtendedHeaders
 // from the network.
 type Exchange interface {
-	// Start starts Exchange.
-	Start(context.Context) error
-	// Stop stops Exchange.
-	Stop(context.Context) error
```

Comment:
Are these moved to another interface or just left in the implementation only?

Reply:
In the implementation only